iset.mm

```diff
@@ -396,13 +454,11 @@ compressed proof steps and less on the length of the label section (since the
     unambiguous 'klr 5';
   $)
 
-  $( Declare typographical constant symbols that are not directly used
-     in the formalism, but *are* symbols we find useful when
-     explaining the formalism. It is much easier to consistently use
-     a single typographical system when generating text. $)
+  $( Declare typographical constant symbols that are not directly used in the
+     formalism but are useful to explain it in comments. $)
 
-  $c & $. $( Ampersand (read: "and-also") $)
-  $c => $. $( Big-to (read: "proves") $)
+  $c & $.  $( Ampersand (read: "and"). $)
+  $c => $.  $( Double right arrow (read: "implies"). $)
```

Comment:
Some authors also use "entails". I think "and" is better that "and-also", and "proves" is not good. I find implies better and not too confusing. But if you think this may confuse new readers (since indeed we have "formula-level implication" ->), then maybe we can remove them altogether.

Reply:
I'd go for "remove them altogether" at least here.

Reply:
To locate it, I searched "ampersand", and ran into https://us.metamath.org/mpeuni/conventions.html where the double right arrow is also given as "implies".

Reply:
Heh. I suppose given how many words we spend on this topic both in that conventions page and sections on "deduction theorem" and so on, a single choice of word here isn't going to make or break anything. So I guess I don't feel super strongly.